app/views/kaminari/_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<li class='<%= "govuk-pagination__item #{' govuk-pagination__item--current' if page.current?} " %>'>
```

Comment:
Thinking about the path of this file do we really want it to be linked to the name "kaminari"? That to me is an unnecessary dependency - maybe we just name it:

`app/views/pagination/_page.html.erb`

That way its agnostic to the underlying implementation.